ui/app/registration/controllers/visitController.js

```diff
@@ -163,17 +163,23 @@ angular.module('bahmni.registration')
             };
             //End :: Registration Page validation
 
-            var reload = function () {
-                $state.transitionTo($state.current, $state.params, {
-                    reload: true,
-                    inherit: false,
-                    notify: true
-                });
+            var afterSave = function () {
+                var forwardUrl = appService.getAppDescriptor().getConfigValue("afterVisitSaveForwardUrl");
+                if (forwardUrl != null) {
+                    $location.url(appService.getAppDescriptor().formatUrl(forwardUrl, {'patientUuid': patientUuid}));
```

Comment:
Got it, thanks.  Just to clarify, is there an existing pattern/method I should be using? I see that in the example you linked to in the previous pull request was using $window.location.href. Is that what I should be using here? Or is a new issue specific to this use case that I will need to work around?  Thanks!